mlflow/R/mlflow/R/docker-utils.R

```diff
@@ -0,0 +1,214 @@
+#' Build a MLflow Docker image
+#'
+#' Build a MLflow Docker image that will an RFunc MLflow model
+#'
+#' @template roxlate-model-uri
+#' @param image_name Name of the Docker image
```

Comment:
If we want to follow the tidyverse style guide, each argument ends with a period, e..g:
```r
#' @param image_name Name of the Docker image.
```
Maybe we could adapt the PR to match this style?